task/07-yield-tasks.js

```diff
@@ -105,11 +105,11 @@ function* depthTraversalTree(root) {
      
     let stack = [];
     yield root;
-    if(root.children){
+    if( root.children ){
```

Comment:
Here and below : you didn't understand correctly 
Spaces after for, if ,while
Curly brace after space